openslides/utils/rest_api.py

```diff
@@ -30,6 +32,69 @@
 router = DefaultRouter()
 
 
+class IdManyRelatedField(ManyRelatedField):
+    field_name_suffix = '_id'
+
+    def bind(self, field_name, parent):
+        self.source = field_name[:-len(self.field_name_suffix)]
+        super().bind(field_name, parent)
+
+
+class IdPrimaryKeyRelatedField(PrimaryKeyRelatedField):
+    """
+    Field, that renames the field name to FIELD_NAME_id.
+
+    Only works together the our ModelSerializer.
+    """
+    field_name_suffix = '_id'
+
+    def bind(self, field_name, parent):
+        """
+        Called when the field is bound to the serializer.
+
+        Changes the source  so that the original field name is used (removes
+        the _id suffix).
+        """
+        if field_name:
+            self.source = field_name[:-len(self.field_name_suffix)]
+        super().bind(field_name, parent)
+
+    @classmethod
+    def many_init(cls, *args, **kwargs):
+        """
+        Method from rest_framework.relations.RelatedField That uses our
+        IdManyRelatedField class.
+        """
+        # TODO: this method can be removed if this is merged and released:
```

Comment:
The issue and pull request were refused so you can remove this comment and may be add a short explanation, that this code is copied ....